ext/fetch/26_fetch.js

```diff
@@ -591,4 +591,38 @@ function handleWasmStreaming(source, rid) {
   }
 }
 
-export { fetch, handleWasmStreaming };
+/**
+ * @param {string} url
+ * @param {BodyInit | null} data
+ */
+async function sendBeacon(url, data = null) {
```

Comment:
**issue:** Spec states
> The [sendBeacon()](https://w3c.github.io/beacon/#dom-navigator-sendbeacon) method returns true if the user agent is able to successfully queue the data for transfer. Otherwise it returns false.

This function returns a Promise that resolves with true or false.

Reply:
Given how Deno expects and enforces resource sanitisation from users, I don't think this API as spec'd is a good fit for the CLI.

Reply:
I don't think resource sanitisation should be considered a blocking issue for this API.